crates/providers/kv-vault/wit/provider-kv-vault.wit

```diff
@@ -0,0 +1,5 @@
+package wasmcloud:provider-kv-vault;
+
+world provider-kv-vault {
+    import wasmcloud:keyvalue/key-value;
```

Comment:
still aware that this is "backwards" -- will eventually get to refactoring this bit out to export rather than import soon.